example/pubspec.lock

```diff
@@ -205,8 +205,8 @@ packages:
   flutter_reactive_ble:
     dependency: "direct main"
     description:
-      path: "../packages/flutter_reactive_ble"
-      relative: true
+      path: "/home/sjoerd/source/flutter_reactive_ble/example/../packages/flutter_reactive_ble"
```

Comment:
I don't think you username should be here

Reply:
There are more entries like this below.

Reply:
.lock files should be ignored in the gitignore.
.lock files will be generated on a new build.

Please take a look at the 'packages/reactive_ble_mobile/android/src/main/kotlin/com/signify/hue/flutterreactiveble/channelhandlers/CharNotificationHandler.kt' file.

Reply:
Maybe It's a good idea to update the .gitignore in the project root?

Reply:
pubspec.lock need to be committed

Reply:
Please read [What not to commit](https://dart.dev/guides/libraries/private-files).

Lock-files for libs, as usual, should not be committed, but for the apps they should be. There should be no lock-files for the libs in the repo, but for the example app there should be.

@sjoerd0301 Make sure you comply with the guidelines regarding this.

Reply:
Deleted .lock files.
Please take a look at the kotlin code, as that _does_ fix a problem

Reply:
You deleted more lock-files, then necessary. The example app is an _app_, lock-files for the apps are to be committed.

A pull request gets merged in its entirety, if one file is good, but others are not, it still doesn't work.